tests/test_functional.py

```diff
@@ -159,10 +160,10 @@ def test_obal_release_upstream_hello_nowait():
 
 @obal_cli_test(repotype='upstream')
 def test_obal_nightly_upstream_hello():
-    assert_obal_success(['nightly', 'hello'])
+    assert_obal_success(['nightly', 'hello', '--source', os.path.join(MOCK_SOURCES_DIR, 'hello-2.10.tar.gz'), '--githash', '0123456789abcdef'])
 
     expected_log = [
-        "tito release dist-git-jenkins -y --arg jenkins_job=hello-master-release",
+        "tito release dist-git -y --arg jenkins_job=hello-master-release",
```

Comment:
If you pass in the explicit source path it surprises me we still pass in the jenkins_job arg. I'd also like to verify in some way we actually pass in the git hash in the version but not sure how.

Reply:
the jenkins_job arg is sitll "passed" in because the variable still exists in the package_manifest. the `build_package` role still populates the variable, but only the `-jenkins` releasers now how to deal with that arg, so it's a no-op.